src/hooks/useDrag.tsx

```diff
@@ -170,7 +223,8 @@ const updateHomepageState = (
   )
     return homepageState
 
-  switch (type) {
+  const typeArray = type.split("-")
+  switch (typeArray[0]) {
```

Comment:
sadly this seems really finicky and prone to error. it appears that we only need the initial element so maybe we should just omit the `index` from `textcard-{index}`?

Reply:
As discussed offline - this issue is due to textcards not being unique, so they're the first section in homepage that requires both `index` and `subindex` to uniquely identify! Will be fixed in a followup PR as discussed by standardizing the type to `textcardcard-{index}` and swapping to if/else instead of switch/case